.github/actions/setup/action.yml

```diff
@@ -29,6 +29,10 @@ runs:
           **/node_modules/
           !node_modules/.cache/nx/
         key: npm-${{ hashFiles('package-lock.json') }}-${{ hashFiles('utils/atomic-storybook/package-lock.json') }}
+    - uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4
+      with:
+        path: packages/quantic/force-app/main/default/staticresources
+        key: quantic-${{ hashFiles('package-lock.json') }}-${{ hashFiles('utils/atomic-storybook/package-lock.json') }}
```

Comment:
The setup action seems to be used in other workflows as well like `e2e-quantic.yml`, `prbot.yml`, `release.yml`, etc.

Do you think the current solution could impact those as well by adding the static resources? For this reason I might be tempted to say maybe we should favor running `npx nx run quantic:build` in the failing task instead so we know it only affects one thing. 

Curious to hear what you think, do you think adding a step to build quantic there would make the task that much slower?

Reply:
This step I'm adding only make use of the NX cache to store the static resources to avoid building them every time. which means that the impact the this step could have is only positive and will win us time.

I don't see any negative impact of using what's already in the cache.